### PR TITLE
Falcon 2273

### PR DESCRIPTION
**Pull request by vrangan**
Added to code to disable external entity resolution and not log messages with info about location of password files

### CODE REVIEW

File: common-types/src/main/java/org/apache/falcon/entity/v0/Entity.java [hunk 1]
```diff
@@ -82,7 +84,11 @@ public String toString() {
     public static Entity fromString(EntityType type, String str) {
         try {
             Unmarshaller unmarshaler = type.getUnmarshaller();
-            return (Entity) unmarshaler.unmarshal(new StringReader(str));
+            XMLInputFactory xif = XMLInputFactory.newFactory();
+            xif.setProperty(XMLInputFactory.IS_SUPPORTING_EXTERNAL_ENTITIES, false);
```
Comment (sandeepSamudrala) [inline common-types/src/main/java/org/apache/falcon/entity/v0/Entity.java]: Can you extract this editing before giving back to user to a method and so can be reused all places.

**Comment by pallavi-rao** [comment]
@vrangan, can you please address or respond to @sandeepSamudrala's comment. I would like this fix to be part of our upcoming 0.11 release.

**Comment by vrangan** [comment]
@pallavi-rao and @sandeepSamudrala 
Updated with review comments.  Please check

**Comment by pallavi-rao** [comment]
Unrelated build failure, merging.

**Pull request closed without merging**